iep-module-userservice/src/main/java/com/netflix/iep/userservice/CompositeUserService.java

```diff
@@ -56,10 +59,12 @@
   }
 
   @Override public String toValidEmail(String email) {
-    for (UserService service : services) {
-      String v = service.toValidEmail(email);
-      if (v != null) {
-        return v;
+    if(email != null) {
```

Comment:
s/if(/if (/

Should we allow null values to be passed in? Maybe this should be a NullPointerException.